dev-config/PSQL_CONNECTION_STRING

```diff
@@ -1 +1 @@
-postgresql+psycopg2://progcom:progcom@localhost:5432/progcom
+postgresql+psycopg2://test:test@localhost:5432/test
```

Comment:
I had to change it on my machine to make it work, but I can exclude this change from the PR.